Projects/SelectTier/Sources/Service/DemoImplementation/SelectTierClientDemo.swift

```diff
@@ -0,0 +1,24 @@
+import hCore
+import hGraphQL
+
+public class SelectTierClientDemo: SelectTierClient {
+    @Inject var octopus: hOctopus
+
```

Comment:
no need for octopus here and hGraphQL at all. You probably switched it with SelectTierClientOctopus xD